robotbase/src/main/java/com/team2357/frc2022/controls/GunnerControls.java

```diff
@@ -141,6 +149,11 @@ public GunnerControls build() {
                 m_gunnerControls.m_xButtonAndLeftDPad.whileActiveOnce(new IntakeTogglePivotCommand(m_intakeSub));
             }
 
+            if (m_shooterSub != null) {
+                m_gunnerControls.m_rightTrigger.whileActiveOnce(
+                        new ShooterSetRPMsCommand(m_shooterSub, 2250, 2750));
```

Comment:
Not sure we want magic numbers here, but I'm guessing this is just for test, right? Maybe add a // TODO: Comment above it?